gradle.properties

```diff
@@ -28,6 +28,7 @@ common_add_to_javadoc=true
 forge_version=44.1.23
 forge_ats_enabled=false
 forge_extra_logging=false
+systemProp.net.minecraftforge.gradle.check.gradle=false
```

Comment:
It is my understanding that Forge offers zero support for running their plugin on unsupported gradle versions, this is not something I want to ship to end users.

Reply:
This completely is temporary and will be removed once out of draft state (definitely after ForgeGradle 6.0 release). I added this to keep my for up to date and see of any issues arise with Gradle 8